src/front-door/azext_front_door/_params.py

```diff
@@ -129,15 +129,17 @@ def load_arguments(self, _):
     with self.argument_context('network front-door routing-rule', arg_group=None) as c:
         c.argument('accepted_protocols', nargs='+', help='Space-separated list of protocols to accept. Default: Http')
         c.argument('patterns_to_match', options_list='--patterns', nargs='+', help='Space-separated list of patterns to match. Default: \'/*\'.')
-        c.argument('rules_engine', help='Name or ID of a Rules Engine configuration.', validator=validate_rules_engine)
+        c.argument('rules_engine', help='Name or ID of a Rules Engine configuration. To unlink property, \"--remove rulesEngine\"', validator=validate_rules_engine)
```

Comment:
Is there an argument "--remove" for the commands? if not, please don't use "--".

Reply:
There is --remove for the command hence to remove property I have mentioned the same in help as it is used to link or unlink rulesEngine resource.